container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -1408,12 +1412,31 @@ instance_groups:
           privileged: true
   configuration:
     templates:
+      properties.cc.ca_cert:                    ((INTERNAL_CA_CERT))
+      properties.cc.cert:                       ((RLP_GATEWAY_CERT))
+      properties.cc.common_name: api-group.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))
+      properties.cc.key:                        ((RLP_GATEWAY_CERT_KEY))
+      properties.http.cert:                     ((RLP_GATEWAY_CERT))
+      properties.http.key:                      ((RLP_GATEWAY_CERT_KEY))
+      properties.logs_provider.ca_cert:         ((INTERNAL_CA_CERT))
+      properties.logs_provider.client_cert:     ((SYSLOG_RLP_CERT))
+      properties.logs_provider.client_key:      ((SYSLOG_RLP_CERT_KEY))
+      #properties.logs_provider.common_name:    "reverselogproxy"
```

Comment:
Why is this commented out? It should either be uncommented, or removed completely.

Reply:
Will remove (was simply a reminder of the default in the spec, and that this property need not be set)

Reply:
Done with commit e7f3d54339f2021d8795054b2f97886a71f100b6